portal-impl/src/com/liferay/portal/util/PropsUtil.java

```diff
@@ -367,8 +379,6 @@ private static String _getLibDir(Class<?> clazz) {
 
 		// Portal shielded container lib directory
 
-		String portalShieldedContainerLibDir = _getLibDir(PropsUtil.class);
-
```

Comment:
Removing _getLibDir for portalShieldedContainerLibDir, this was using in DBBuilder.
We moved this part of logic into InitUtil.
If this is not set, ant all throws compile error